tasks/update-granules-cmr-metadata-file-links/index.js

```diff
@@ -41,15 +41,24 @@ async function updateEachCmrFileAccessURLs(
   cmrGranuleUrlType,
   distEndpoint,
   bucketTypes,
-  distributionBucketMap
+  distributionBucketMap,
+  excludeFileRegexPattern
 ) {
   return await Promise.all(cmrFiles.map(async (cmrFile) => {
     const granuleId = cmrFile.granuleId;
     const granule = granulesObject[granuleId];
+    let files = granule.files;
+    if (excludeFileRegexPattern !== null) {
+      const excludeFileRegex = new RegExp(excludeFileRegexPattern);
+      files = granule.files.filter((file) => !file.key.match(excludeFileRegex));
+      if (files.length === granule.files.length) {
+        throw new Error(`No files matched the excludeFileRegex ${excludeFileRegexPattern}.  Found files: ${files.map((file) => file.key).join(', ')}`);
```

Comment:
Is this any error?  If no file matches the `excludeFileRegexPattern`, do all the granules have to have the DMRPP file?

Reply:
I don't think so? I'm not sure what you mean. files will be the remaining (non-filtered) files, and it throws an error if you have set a "bad" regex (no files were filtered) I think I want to change that to a warning, rather than an error

Reply:
Yeah, I also prefer to log a warning rather than an error, so the workflow can be used by granules which don't have the files matching excludeFileRegexPattern

Reply:
https://github.com/nasa/cumulus/pull/3791/commits/1f7e9cc89fbcd6439e1f11726a83f719b3a4a266
yup, fixed it
and confirmed with mike dorfman that switching to logging wouldn't interfere with their use-case